### ISSUE TITLE
Using a template engine

### ISSUE THREAD

**Issue by white-gecko** [MyProfile/myprofile#10]
Hey Andrei,
as I've been looking through the code, I saw much HTML inside the PHP. Did you consider using some kind of template engine?

Kind regards
Nate

**Comment by deiu** [comment]
Yes, I would very much like to switch to a template-based UI, but I had to cut some corners to be able to release the code in time for WWW2012. What I would actually like to do is to add a HTML UI layer and a linked data "UI" layer. All data that is visible for human eyes should also be made available through linked data.

Maybe I can merge this issue into the first milestone. How does that sound?

**Comment by deiu** [comment]
If you don't mind, I'm going to move this issue to the "Add RDF markup for HTML pages." milestone.
